### PR TITLE
SSL Pinning Blog Post

### PR DESCRIPTION
**Pull request by erisu**
(05/23/2017) SSL Pinning Blog Post

### CODE REVIEW

File: blog/posts/2017-05-23.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+author: bryan
+date: 2017-05-23
+id: enhanced-cordova-app-security-with-ssl-certificate-pinning
+title: "Enhanced Cordova App Security with SSL Certificate Pinning"
+product: monaca
+category: development
+tags: monaca, cordova, ssl
+---
+
+Simple static application has fewer security concerns compared to dynamic application that requires an internet connection. An example concern, that a developer might have with either a static or dynamic application is how to prevent non-authorized users from copying or reading source code. This can easily be resolved by encrypting to the application content.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: `A` simple static application .... compared to `a` dynamic application ... `the` source code.

File: blog/posts/2017-05-23.markdown [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+More security concerns appear when an internet connection is established for fetch data. This connection can put a user or application at risk. The biggest concerns, either hybrid or native development, is the data integrity. Did the data come from the originated source? Has the data been tampered with before reaching the app? This commonly known as a man in the middle attack.
+
+<!-- more -->
+
+# Requirements
+- Monaca account with a [plan](https://monaca.io/pricing-detail.html) that supports `Custom Cordova plugin`.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: `A` Monaca account

File: blog/posts/2017-05-23.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+If the fingerprint is valid, the success callback is executed. This app example will perform the second request to fetch the actual data for display when the success callback is executed.
+
+If the fingerprint is invalid, there are four possible likely outcomes.
+- The URL has a new public key with a new fingerprint.
+- The fingerprint in code was inserted incorrectly.
+- The connection to the server was lost or response timeout.
+- Possible Man in the Middle Attack.
+
+Since this plugin will only performs pre-checks before making the official request for the response data, it will mitigate to an extent but contains some drawbacks.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: `perform`

File: blog/posts/2017-05-23.markdown [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Drawbacks:
+- If you require to check on every request, this plugin would have to duplicate each request.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: require `checking` should be the right form I think

File: blog/posts/2017-05-23.markdown [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+- If the check occurs at a given time during the apps lifecycle, for example start up, the connection maybe secure at that give time but not always. For example: your wifi connection changes networks and your app was already started.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: `app's lifecyrcle`... `may be secure` ... 

Maybe `,` is better than `:` after `For example`.

File: blog/posts/2017-05-23.markdown [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+Some of these drawbacks could also be mitigated but the idea case would be to test on every request and not duplicate the requests.
```
Comment (andipavllo) [inline blog/posts/2017-05-23.markdown]: `ideal case`

**Comment by andipavllo** [comment]
thank you @erisu !
